examples/importers/runtests.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+DATA=../importers_tests
+python3 acme_pdf.py   test $DATA/acme_pdf
```

Comment:
This should be `python3 acme_pdf.py   test $DATA` ideally.

Reply:
Also, while we are at it, can we name the importers `acme.py` and `utrade.py`?

Reply:
Yep. I like it
I always have multiple formats in my real ones somehow, but this does make the examples simpler.
Will update now.

Reply:
done